swarm/network/stream/delivery_test.go

```diff
@@ -321,11 +321,12 @@ func testDeliveryFromNodes(t *testing.T, nodes, conns, chunkCount int, skipCheck
 	defaultSkipCheck = skipCheck
 	toAddr = network.NewAddrFromNodeID
 	conf := &streamTesting.RunConfig{
```

Comment:
I am not 100% sure that this is the right structure to plug `EnableMsgEvents` to, but this seems to be the configuration for the test and felt somewhat appropriate place.

Reply:
looks about right to me